.github/workflows/ci.yml

```diff
@@ -22,6 +22,7 @@ jobs:
         version:
           - '1.6'
           - '1'
+          - 'nightly'
```

Comment:
I don't like testing on nightly since its a moving target and it adds too much burden to merging PRs.

Reply:
Sure, I'll revert that. This was to show that this package will start to fail from v1.9-alpha+ due to some allocation issue that is unrelated to this PR.